client/src/components/dashboard/Community/UserCard.js

```diff
@@ -71,8 +91,10 @@ class UserCard extends React.Component {
 
     const {
       user,
+      user: {
```

Comment:
@no-stack-dub-sack but you effectively are putting in the key `user` twice here aren't you? looks like invalid json, but I may not be following the logic.